sdk/servicebus/service-bus/test/utils/managementUtils.ts

```diff
@@ -10,12 +10,13 @@ import {
 
 import { EnvVarKeys, getEnvVars } from "./envVarUtils";
 
-export async function recreateQueue(queueName: string, parameters: QueueOptions): Promise<void> {
-  const env = getEnvVars();
-  const client = await new ServiceBusAtomManagementClient(
-    env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]
-  );
+const env = getEnvVars();
+let client: ServiceBusAtomManagementClient;
+(async function() {
+  client = await new ServiceBusAtomManagementClient(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]);
+})();
```

Comment:
The function is being called, but not awaited.
So there is no guarantee, that the client would be ready for use by the methods in this file.

I would suggest making this a proper function with a name, say  `getManagementClient` and add a line to return the `client` if it exists, else create it. Then call this method from each of the "recreate" functions.

Reply:
Oh right, didn't show up in local test runs - would have if there was a network delay
Fixed